### DISCUSSION TITLE
Capability to Import from existing Database

### DISCUSSION THREAD

**Discussion by jagadishbabu** [Sunbird-RC/community#8] [Ideas & Enhancements]
Sunbird-RC team -  Please consider bringing in the capability to Import into Registry from existing databases with due validation controls - So that Registry initiation does not need fresh enrollments (only). It saves cost.  I am however assuming that records from existing databases are not completely trustworthy.

**Comment by jagadishbabu** [comment]
Sunbird-RC team- i don't see any update on this thread. request to look into this request.

**Comment by parthlawate** [comment]
@jagadishbabu We've updated the documentation on how existing databases can be used without compromising core registry principles https://docs.sunbirdrc.dev/use/leveraging-existing-data-stores-directories  .. Hope this helps ! 

@pramodkvarma do comment if you see more ways of achieving this

**Reply by rahul101001000** [reply]
Returns 404, could you please double check the provided URL @parthlawate Thanks in advance!

**Reply by pramodkvarma** [reply]
Thanks Rahul. Link updated above.

**Reply by rahul101001000** [reply]
Thanks a lot Pramod, very relevant doc.

**Comment by pramodkvarma** [comment]
Good start. But, need to make it better so that it is useful to intended audience. If business person, bring the capability and benefits out. If tech person, bring details on how to do this. So, I suggest we have two sections in this doc - what and why (capability/benefit) and then how (for tech person to understand how to do).

**Reply by parthlawate** [reply]
I've tweaked it a little & changed the structure to add the Why & How .. The What i've brought out in the title and description now..  @kshitij can you take another pass at this to improve the same ?

**Comment by pramodkvarma** [comment]
Nice! For now, leave it at that. Docs can always be improved later. Thanks.

**Comment by parthlawate** [comment]
Works.. Will mark as answered for now.. As we get more real usecases, we will revisit this & also add more technical docs around it
